### PR TITLE
intro-to-pandas

### PR DESCRIPTION
**Pull request by scotgopal**
Please do review the Pandas content.

### CODE REVIEW

File: Pandas/Exercise/01 - Fundamentals of Pandas.ipynb [hunk 1]
```diff
@@ -0,0 +1,582 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4fb51a03",
+   "metadata": {},
+   "source": [
+    "Note that when importing a well-formatted `.json` file, the indexes have been inferred correctly without any need to specify in the attributes. You may also supply URLs into the read methods.\n",
+    "\n",
+    "`pandas` also offers methods to read from binary files which are not human-readable such as `.pkl`. Different read methods have different attributes to them to deal with importing issues that come up when reading data from their respective extensions. Hence, its a no-brainer to use the right method for the given file format. "
```
Comment (KianYang-Lee) [inline Pandas/Exercise/01 - Fundamentals of Pandas.ipynb]: its => it's ?

**Review by evanyeohboonkhai — APPROVED** [review]
@scot-certifai Thanks for the effort of creating the notebook. All good and error free. I very enjoy reading it.

File: Pandas/Exercise/01 - Fundamentals of Pandas.ipynb [hunk 2]
```diff
@@ -0,0 +1,582 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7ffc62b5",
+   "metadata": {},
+   "source": [
+    "Observe that `pandas` provides a great number of options to read data files from various formats. Another common format of storing data is in the form of `.json` files.\n",
+    "\n",
+    "### <a name=\"read_json\">Read from JSON</a>\n",
+    "Big data sets in database systems are often extracted or stored in JSON files. JSON is a plain text (which is human-readable by the way) and is a popular standard for communicating between client and servers.\n",
```
Comment (KianYang-Lee) [inline Pandas/Exercise/01 - Fundamentals of Pandas.ipynb]: "JSON  is a plain text (...) **file** ?

File: Pandas/Exercise/03 - Common Operations.ipynb [hunk 3]
```diff
@@ -0,0 +1,703 @@
...
+    "Amazing. As you progress in your abilities to filter, you will be able to answer more and more complex questions, critical to your analysis. That is why good data scientists must not only know to code, but must also possess the creativity to ask the right questions.\n",
+    "\n",
+    "##  <a name=\"summary\">Summary</a>\n",
+    "To conclude, you should now be able to:\n",
+    "\n",
+    "1. Understand the difference between `.iloc` and `.loc`.\n",
+    "2. Slice and select your data through index, labels and conditions.\n",
+    "3. Obtain data by columns and by rows.\n",
+    "\n",
+    "Congratulations, that concludes this lesson. In the next tutorial, we'll learn on how to cleanup our data and deal with missing values.\n",
```
Comment (KianYang-Lee) [inline Pandas/Exercise/03 - Common Operations.ipynb]: cleanup => clean up ?

File: Pandas/Exercise/04 - Data Cleanup and Missing Data.ipynb [hunk 4]
```diff
@@ -0,0 +1,502 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "referenced-perspective",
+   "metadata": {},
+   "source": [
+    "## <a name=\"description\">Notebook Description</a>\n",
+    "\n",
+    "When you get out of the classroom and start applying your analytical skills in the real world, you'll realise that the dataset that you obtain is not really like how you had expected. Datasets go through varieties of processes before reaching your hands, causing it to end up being unstructured and rather 'unfriendly' for the data scientist. This tutorial focuses on how to clean up your data and a handful of methods of dealing with missing data.\n",
```
Comment (KianYang-Lee) [inline Pandas/Exercise/04 - Data Cleanup and Missing Data.ipynb]: handful of methods **of** => **in** dealing with data ?

File: Pandas/Exercise/04 - Data Cleanup and Missing Data.ipynb [hunk 5]
```diff
@@ -0,0 +1,502 @@
...
+   "source": [
+    "##  <a name=\"summary\">Summary</a>\n",
+    "To conclude, you should now be able to:\n",
+    "\n",
+    "1. List, rename and delete columns.\n",
+    "2. Detect and count `NaN` values\n",
+    "3. Remove rows with `NaN` values.\n",
+    "4. Replace `NaN` values.\n",
+    "\n",
+    "Congratulations, that concludes this tutorial. Exploring, cleaning and transforming daata is an essential skill in data science. After some practice, you should be really comfortable with most of the basics. \n",
```
Comment (KianYang-Lee) [inline Pandas/Exercise/04 - Data Cleanup and Missing Data.ipynb]: typo **daata** => data

File: Pandas/Exercise/04 - Data Cleanup and Missing Data.ipynb [hunk 6]
```diff
@@ -0,0 +1,502 @@
...
+    "\n",
+    "In your learning journey, you may come across errors in all different forms. Don't let that discourage you as even the best programmers have them too. Steer through the error by interpreting it and try your level best to debug your code. You may also use this [guide](https://geo-python.github.io/site/notebooks/L6/errors.html) as a point of reference.\n",
```
Comment (KianYang-Lee) [inline Pandas/Exercise/04 - Data Cleanup and Missing Data.ipynb]: **all** => **many** different forms ?

**Review by KianYang-Lee — APPROVED** [review]
Just some typos and some grammatical suggestions, please do the necessary thanks have left the suggestions

File: Pandas/Exercise/01 - Fundamentals of Pandas.ipynb (hunk 1 above)
Comment (scotgopal) [reply Pandas/Exercise/01 - Fundamentals of Pandas.ipynb]: Corrected. Thanks for the flag.

**Comment by scotgopal** [comment]
Done the corrections and its ready for merging. @KianYang-Lee

**Pull request merged**